src/app/container/container.component.html

```diff
@@ -13,7 +13,7 @@
   ~    See the License for the specific language governing permissions and
   ~    limitations under the License.
   -->
-<div *ngIf="tool; else workflow">
+<div *ngIf="tool || !displayAppTool; else workflow">
```

Comment:
The template and property names were like this already, but this code makes it seem like there are 3 entity types to handle: Tool, AppTool, and Workflow. But this component handles just two types, Tool and AppTool, even though AppTool extends Workflow.

Suggest: change the template id to `appTool`, i.e., `<div *ngIf="tool || !displayAppTool; else appTool">`